server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/AdmissionControlService.java

```diff
@@ -128,13 +141,31 @@ public AdmissionController getAdmissionController(String controllerName) {
      */
     public AdmissionControlStats stats() {
         List<AdmissionControllerStats> statsList = new ArrayList<>();
-        if (this.admissionControllers.size() > 0) {
+        if (!this.admissionControllers.isEmpty()) {
             this.admissionControllers.forEach((controllerName, admissionController) -> {
-                AdmissionControllerStats admissionControllerStats = new AdmissionControllerStats(admissionController);
-                statsList.add(admissionControllerStats);
+                if (controllerName.equals(IO_BASED_ADMISSION_CONTROLLER)) {
```

Comment:
if you dont register as mentioned above, this logic we wont have to modify.

Reply:
Agree will update